signingscript/tox.ini

```diff
@@ -21,7 +21,7 @@ deps=
     py37: -rrequirements/test.txt
 
 commands =
-    py.test --cov-config .coveragerc --cov=signingscript --cov-append --cov-report term-missing {posargs}
+    py.test --cov-config .coveragerc --cov=signingscript --cov-append --cov-report term-missing {posargs} --pdb
```

Comment:
is this change intentional?

Reply:
also, for your local use, you can do the equivalent by running `tox -e py37 -- --pdb`

Reply:
this no longer seems to work locally, fwiw.  (I tried `tox -e py38 -- --pdb` and it tried to run a pdb command and nothing more)

Reply:
IIRC, tox.ini changed when Ben landed the standardization patches?

Reply:
No worry in the end, here, was just a passing comment